src/ThermalScattering.tex

```diff
@@ -1,49 +1,80 @@
 %!TEX root = ../ACEFormat.tex
 \section[Thermal Scattering]{Thermal Scattering \SaB}\label{sec:ThermalScattering}
 
-Data from thermal \SaB\ tables provide a complete representation of thermal neutron scattering by molecules and crystalline solids. Cross sections for (coherent and incoherent) elastic and (incoherent) inelastic scattering are found on the tables. A coupled energy/angle representation is used to describe the spectra of inelastically scattered neutrons. Angular distributions for elastic scattering are also provided.
+Data from thermal \SaB\ tables provide a complete representation of thermal neutron scattering by molecules
```

Comment:
I'm not a big fan of breaking up the lines like this here, or elsewhere in the document. I suppose that's a function of the text editor being used, but it makes many more differences pop up that are not really differences.